cli/lib/commands/ServeCommand.dart

```diff
@@ -0,0 +1,32 @@
+part of glacier_cli;
+
+class ServeCommand extends Command {
+  @override
+  String get description => "Run the builtin development server";
+
+  @override
+  String get name => "serve";
+
+  ServeCommand() {
+    argParser.addOption("port", defaultsTo: "3000", valueHelp: "3001");
+  }
+
+  @override
+  Future<void> run() async {
+    final config = GlacierConfig.loadFromFile();
+    final destinationDir = path.join(
+      Directory.current.absolute.path,
+      config.destinationDirectory.replaceAll("./", ""),
+    );
+    final port = int.tryParse(argResults!["port"] as String)!;
+
+    final handler = shelf_static.createStaticHandler(
+      destinationDir,
+      defaultDocument: "index.html",
+    );
+
+    await shelf_io
+        .serve(handler, "localhost", port)
```

Comment:
if we are going to allow spcifying port maybe good idea would be host too?

Reply:
done